### PR TITLE
Updated use of UploadLicense

### PR DESCRIPTION
**Pull request by MrAsler**

### CODE REVIEW

**Comment by pjft** [comment]
Thank you @MrAsler . We saw you still updating the PR today - let us know when it's ready for review, and keep them coming!

File: src/setup/unattended-install/config-tool-ref.md [hunk 1]
```diff
@@ -80,28 +105,28 @@ ConfigurationTool.com {/SetupInstall <platform_db_admin_username> <platform_db_a
...
     `[<enable_https>]` is an optional boolean argument; if this argument is not provided the setting will remain unchanged. If the string "true" (case insensitive) is provided, the Enable HTTPS setting is set to "true"; if any other string is provided the setting Enable HTTPS is set to "false". The applied value of the Enable HTTPS setting is displayed.
 
 `/CreateUpgradeCacheInvalidationService`
 
 :   Installs cache invalidation service or reconfigures the service given in the configuration file (`server.hsconf`).
-
-`/SetPlatformServerAdminPassword <platform_server_admin_password>`
-
-:   Defines the password for the Platform Server Admin user, if the user is active.  
-    Note: This command does not work when using Integrated Authentication.
```
Comment (pedrosousa) [inline src/setup/unattended-install/config-tool-ref.md:104]: We're dropping this sentence... Should we keep it in the new location (`/UpgradeInstall`)?

File: src/setup/unattended-install/config-tool-ref.md [hunk 2]
```diff
@@ -24,33 +24,46 @@ The Configuration Tool command line returns non-zero values when an error occurs
...
+    | /CreateUpgradeCacheInvalidationService
     | /EnableServerAPI
     | /DisableServerAPI
+    | /UpgradeEnvironment
 ```
 
 ## Parameters
 
-`/SetupInstall [<platform_db_admin_username> <platform_db_admin_password>] [<logging_db_admin_username> <logging_db_admin_password>]`
+`/SetupInstall [<platform_db_admin_username>] [<platform_db_admin_password>] [<logging_db_admin_username>] [<logging_db_admin_password>] [/SetPlatformServerAdminPassword platform_server_admin_password]`
```
Comment (pedrosousa) [inline src/setup/unattended-install/config-tool-ref.md:50]: This updated syntax is different from the global syntax above:

At the top (syntax summary):

> `/SetupInstall <platform_db_admin_username> <platform_db_admin_password> <logging_db_admin_username> <logging_db_admin_password> [/SetPlatformServerAdminPassword <platform_server_admin_password>]`

vs. here:

> `/SetupInstall [<platform_db_admin_username>] [<platform_db_admin_password>] [<logging_db_admin_username>] [<logging_db_admin_password>] [/SetPlatformServerAdminPassword platform_server_admin_password]`

Can you supply the usernames without the passwords for platform and logging DBs?

Comment (MrAsler) [reply src/setup/unattended-install/config-tool-ref.md:50]: > 
> 
> This updated syntax is different from the global syntax above:
> 
> At the top (syntax summary):
> 
> > `/SetupInstall <platform_db_admin_username> <platform_db_admin_password> <logging_db_admin_username> <logging_db_admin_password> [/SetPlatformServerAdminPassword <platform_server_admin_password>]`
> 
> vs. here:
> 
> > `/SetupInstall [<platform_db_admin_username>] [<platform_db_admin_password>] [<logging_db_admin_username>] [<logging_db_admin_password>] [/SetPlatformServerAdminPassword platform_server_admin_password]`
> 
> Can you supply the usernames without the passwords for platform and logging DBs?

We can install without any arguments, but it does indeed not make much sense if we try to just use some of the arguments.

**Comment by pedrosousa** [comment]
Thanks again, @MrAsler!

**Pull request merged**